configs/deblurganv2/deblurganv2_fpn-mobilenet_1xb1_gopro.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+_base_ = ['../_base_/default_runtime.py']
+
+save_dir = './work_dir/'
+
+model = dict(
+    type='DeblurGanV2',
+    generator=dict(
+        type='DeblurGanV2Generator',
+        backbone='FPNMobileNet',
+        norm_layer='instance',
+        output_ch=3,
+        num_filter=64,
+        num_filter_fpn=128,
+    ),
+    discriminator=dict(
+        type='DeblurGanV2Discriminator',
+        backbone='DoubleGan',
+        norm_layer='instance',
+        d_layers=3,
+    ),
+    pixel_loss=dict(
+        type='PerceptualLoss', layer_weights={'14': 1}, criterion='mse'),
+    disc_loss=dict(type='AdvLoss', loss_type='ragan-ls'),
+    adv_lambda=0.001,
+    warmup_num=3,
+    data_preprocessor=dict(
+        type='DataPreprocessor',
+        mean=[127.5] * 3,
+        std=[127.5] * 3,
+    ))
+
+train_pipeline = [
+    dict(type='LoadImageFromFile', key='img'),
+    dict(type='LoadImageFromFile', key='gt'),
+    dict(type='SetValues', dictionary=dict(scale=1)),
+    dict(type='PairedAlbuTransForms', size=256, lq_key='img', gt_key='gt'),
+    dict(
+        type='AlbuCorruptFunction',
+        keys=['img'],
+        config=[{
+            'name': 'cutout',
+            'prob': 0.5,
+            'num_holes': 3,
+            'max_h_size': 25,
+            'max_w_size': 25
+        }, {
+            'name': 'jpeg',
+            'quality_lower': 70,
+            'quality_upper': 90
+        }, {
+            'name': 'motion_blur'
+        }, {
+            'name': 'median_blur'
+        }, {
+            'name': 'gamma'
+        }, {
+            'name': 'rgb_shift'
+        }, {
+            'name': 'hsv_shift'
+        }, {
+            'name': 'sharpen'
+        }]),
+    dict(type='PackInputs')
+]
+
+val_pipeline = [
+    dict(type='LoadImageFromFile', key='img'),
+    dict(type='LoadImageFromFile', key='gt'),
+    dict(type='SetValues', dictionary=dict(scale=1)),
+    dict(type='PairedAlbuTransForms', size=256, lq_key='img', gt_key='gt'),
+    dict(
+        type='AlbuCorruptFunction',
+        keys=['img'],
+        config=[{
+            'name': 'cutout',
+            'prob': 0.5,
+            'num_holes': 3,
+            'max_h_size': 25,
+            'max_w_size': 25
+        }, {
+            'name': 'jpeg',
+            'quality_lower': 70,
+            'quality_upper': 90
+        }, {
+            'name': 'motion_blur'
+        }, {
+            'name': 'median_blur'
+        }, {
+            'name': 'gamma'
+        }, {
+            'name': 'rgb_shift'
+        }, {
+            'name': 'hsv_shift'
+        }, {
+            'name': 'sharpen'
+        }]),
+    dict(type='PackInputs')
+]
+
+test_pipeline = [
+    dict(type='LoadImageFromFile', key='img'),
+    dict(type='LoadImageFromFile', key='gt'),
+    dict(type='PackInputs')
+]
+
+data_root = 'G:/github/DeblurGANv2/data/gopro/debug/'
```

Comment:
Since we always suggest users to soft link the path to data under the project folder, therefore, we can use relative path here.
```suggestion
data_root = 'data/gopro'
```